apis/fluentd/v1alpha1/tests/expected/fluentd-mixed-cfgs-output-es.cfg

```diff
@@ -7,33 +7,33 @@
   @id  main
   @type  label_router
   <route>
-    @label  @48b7cb809bc2361ba336802a95eca0d4
+    @label  @a2170d34e9940ec56d328100e375c43e
     <match>
-      namespaces  kube-system,kubesphere-monitoring-system
+      namespaces  default,kube-system
     </match>
   </route>
   <route>
-    @label  @4a579b91a4b1f32966ef08e305396a08
+    @label  @2d9e59757d3bfc66d93c3bc44b408922
     <match>
-      namespaces  kubesphere-logging-system
+      namespaces  fluent
     </match>
   </route>
 </match>
-<label @48b7cb809bc2361ba336802a95eca0d4>
+<label @a2170d34e9940ec56d328100e375c43e>
   <match **>
     @id  ClusterFluentdConfig-cluster-fluentd-config::cluster::clusteroutput::fluentd-output-es-0
     @type  elasticsearch
-    host  elasticsearch-logging-data.kubesphere-logging-system.svc
+    host  elasticsearch-logging-data.fluent.svc
```

Comment:
we can keep the es namespace as it is before : elasticsearch-logging-data.kubesphere-logging-system.svc